analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/ProgramStateTest.cs

```diff
@@ -141,6 +142,25 @@ public void SetOperationValue_Overrides()
             sut[operation].Should().Be(value2);
         }
 
+        [TestMethod]
+        public void SetOperationValue_Null_ReturnsNull()
```

Comment:
I would like to be explicit here:
```suggestion
        public void SetOperationValue_NullValue_ReturnsNull()
```